api/graphql/schema.graphql

```diff
@@ -741,7 +740,7 @@ input PoolFilterInput {
   name: String @scope
   team: String @scope
   streams: [PoolStream!] @scope
-  statuses: [PoolStatus!] @scope
+  statuses: [PoolStatus!] = [] @scope
   processNumber: String @scope
```

Comment:
If the field is not present, the scope is not run. Given the scope always needs to be used, even if just to apply the archived filtering, a default must exist